### DISCUSSION TITLE
Update Jekyll

### DISCUSSION THREAD

**Discussion by akibrhast** [hackforla/website#1283] [General] [answered]
### Overview
We need to update jekyll. We are currently using version 3.8.5 . The latest stable version is version 4.2.0.
Why do we need to update?
Because -> [Source](https://jekyllrb.com/docs/upgrading/#:~:text=We%20recommend%20you%20update%20Jekyll,update%20to%20the%20latest%20versions.)
![image](https://user-images.githubusercontent.com/32349001/111851670-83ce6b00-88ea-11eb-8067-34f1ecf07cd7.png)

### Action Items
- [ ] Upgrade `image:` in docker-compose from `image: jekyll/jekyll:pages` -> `image: jekyll/jekyll:stable`

### Resources/Instructions
- [Jekyll Docs](https://jekyllrb.com/docs/upgrading/#:~:text=We%20recommend%20you%20update%20Jekyll,update%20to%20the%20latest%20versions.)
- [Jekyll Stable](https://hub.docker.com/layers/jekyll/jekyll/stable/images/sha256-de110e0fdde342de0cb6fd5c63f64934b6ce25d0dd826c67113b3a5a3370784e?context=explore) 
- [Jekyll Pages](https://hub.docker.com/layers/jekyll/jekyll/pages/images/sha256-a80baa7fc7837a8d7a912c71ba583156a3f97683b1d48bac39e32d2c8c2c974f?context=explore)

**Comment by akibrhast** [comment]
@cnk Your thoughts and opinion on this?

**Comment by cnk** [comment, marked as answer]
We are currently on the highest version supported by GitHub pages - 3.9.0. 

https://pages.github.com/versions/

To find what version we are using, exec into the container and run `gem list jekyll` We see `jekyll (3.9.0, 3.8.5)`. The way Ruby gems works, it will load the latest version, so we are running 3.9.0.

When GH updates its supported versions, we may have to move too, but not before.

**Comment by akibrhast** [comment]
Understood. Thanks! I am going to close this
